components/LiteMintNFT.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="flex flex-col items-stretch grow space-y-4">
+    <UAlert
+      v-if="error"
+      icon="i-heroicons-exclamation-triangle"
+      color="red"
+      variant="soft"
+      :title="`${error}`"
+      :close-button="{ icon: 'i-heroicons-x-mark-20-solid', color: 'red', variant: 'link', padded: false }"
+      @close="error = ''"
+    />
+
+    <UCard
+      class="flex-1"
+      :ui="{ body: { base: 'space-y-4' } }"
+    >
+      <template #header>
+        <h3 class="font-bold">
+          Mint NFT by filling required information
+        </h3>
+      </template>
+
+      <NFTMintForm
+        ref="formRef"
+        v-model="formState"
+        :max-supply="classMaxSupply"
+        :show-max-supply="false"
+      />
+
+      <div v-if="isLoading" class="w-full">
+        <div class="space-y-3">
+          <div class="flex justify-between items-center">
+            <UBadge color="Badge" variant="soft">
+              Minting NFT
+            </UBadge>
+            <p class="text-xs text-gray-500">
+              請勿關閉此視窗，直到操作完成。
+            </p>
+          </div>
+          <UProgress
+            animation="carousel"
+            color="primary"
+            class="w-full"
+          />
+        </div>
+      </div>
+      <div class="flex justify-center">
+        <UButton
+          label="Submit"
+          :loading="isLoading"
+          size="lg"
+          :disabled="isLoading"
+          @click="onClickMintByInputting"
+        />
+      </div>
+    </UCard>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from 'pinia'
+import { v4 as uuidv4 } from 'uuid'
+import { parse } from 'csv-parse/sync'
+import { stringify } from 'csv-stringify/sync'
```

Comment:
Why parse csv here?

Reply:
for `nftCSVData`

Reply:
Just keep it as normal array/json instead of parsing it as csv, if we don't download it as csv?

Reply:
[🔥 Remove csv related code](https://github.com/likecoin/nft-book-press/pull/334/commits/d2e5b72929e22d9e50fd5d66212ae8efc73bc4fc)